src/pages/podcasts.module.scss

```diff
@@ -0,0 +1,43 @@
+.section {
+  &:not(:last-child) {
+    margin-bottom: 4rem;
+  }
+}
+
+.twoColumn {
+  display: grid;
+  grid-template-columns: repeat(2, calc(50% - 1rem));
+  grid-gap: 2rem;
+}
+
+.img {
+  width: 100%;
+}
+
+.podcasts {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 2rem;
+}
+
+.point {
+  text-align: center;
+}
+
+.player {
+  overflow: hidden;
+}
+
+.player iframe {
+  width: 100%;
+  height: 421px;
```

Comment:
It looks like this height is adding a bunch of whitespace between the player and the footer. 

<img width="1296" alt="Screen Shot 2020-07-20 at 9 58 32 PM" src="https://user-images.githubusercontent.com/565661/88014588-2b816800-cad4-11ea-89d7-0673753485bd.png">

It looks like the visible part of the player is `200px` tall, but also looks like there is some kind of invisible content that causes the content to scroll if this is set to `200px`. Do you know if there is a way to configure the player to reduce the height of the player to the visible portion? If possible, it would be great to reduce some of the whitespace here.

Reply:
The iframe has an episode list that we're not seeing as we only have the one episode. For now I've hidden the windows related to the list so I'm able to reduce the height and get rid of the whitespace. It was bothering me as well.